src/eventParsers/parseRdpcProgramUpdateEvent.ts

```diff
@@ -17,7 +17,8 @@ enum RDPC_EVENT_STATE {
 
 type RdpcProgramUpdateEvent = {
   studyId: string;
-  state: RDPC_EVENT_STATE;
+  state: string;
```

Comment:
type of `state` is changed from `RDPC_EVENT_STATE` to string, is there a reason for that?